pep-0622.rst

```diff
@@ -198,6 +198,19 @@ statement. For example::
           ...
   print(x, y)  # This works
 
+During failed pattern matches, some subpatterns may succeed. For example,
+while matching the value ``[0, 1, 2]`` with the pattern ``(0, x, 1)``, the
+subpattern `x` may succeed if the list elements are matched from left to right.
+The implementation may choose to either make persistent bindings for those
+partial matches or not. User code including a `match` statement should not rely
+on the bindings being made for a failed match, but also shouldn't assume that
+variables being unchanged by a negative match. This part of the behavior is
```

Comment:
We don't define "negative match" IIRC, and using it so close to "failed match" suggests it is something different. Once you fix this, please send this to python/peps!

Reply:
Done, thanks.